EpiAware/src/EpiInfModels/InfectionODEProcess.jl

```diff
@@ -0,0 +1,182 @@
+"""
+A structure to hold the initial condition and parameters for an ODE (Ordinary Differential Equation) process.
+`params::ODEParams` is used in the method `generate_latent_infs(epi_model::InfectionODEProcess, params::ODEParams)`
+
+# Constructors
+- `ODEParams(; u0::VecOrMat, p::VecOrMat)`: Create an `ODEParams` object with the initial condition(s) `u0` and parameters `p`.
+
+# Fields
+- `u0::VecOrMat{T}`: The initial condition(s) for the ODE, which can be a vector or matrix of type `T`.
+- `p::VecOrMat{T}`: The parameters for the ODE, which can be a vector or matrix of type `T`.
+
+# Type Parameters
+- `T`: The type of the elements in `u0` and `p`.
+
+# Example
+```jldoctest
+using EpiAware
+params = ODEParams(; u0 = ones(10), p = [2, 3])
+
+# output
+
+ODEParams{Float64}([1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0], [2.0, 3.0])
+```
+"""
+struct ODEParams{T}
```

Comment:
why not use @kwdef?

Reply:
I thought about this, but since I have outer constructors with `kwargs` it seemed pretty unnecessary

Reply:
In fact, just to add clarity. The outer constructor for `ODEParams` also does a `promote_type` step (which is useful for keeping the parametric struct type simple), but it would conflict with the constructor generated by `@kwdef` here.